nebula1/cometbft_config.toml

```diff
@@ -0,0 +1,13 @@
+[p2p]
+pex = true
+seeds = "4567c84f2d62e252d3ee4cbe691e38edb6d72d74@185.198.234.44:26656,00ec120e907bbe238dff50f229200ad3c42f51bb@185.198.234.88:26656"
+persistent_peers = "81427e8bf332e9b26cad5e79a8decbfd69e76677@185.198.234.18:26656,1fe60e0c397c7665b41c99a24da3bfc8d6308ff2@185.198.234.22:26656"
```

Comment:
Why only two here?

Reply:
I think we can remove all the seeds, and set only peers with PEX enabled:

```
All peers relay peers they know of by default. This is called the peer exchange protocol (PeX). With PeX, peers will be gossiping about known peers and forming a network, storing peer addresses in the addrbook. Because of this, you don't have to use a seed node if you have a live persistent peer.
```